systemtest/src/test/java/io/strimzi/systemtest/tracing/TracingAbstractST.java

```diff
@@ -543,26 +545,21 @@ void doTestKafkaBridgeServiceWithHttpTracing(final ExtensionContext extensionCon
      * Delete Jaeger instance
      */
     private void deleteJaeger() {
-        while (!this.jaegerConfigs.empty()) {
-            cmdKubeClient().namespace(this.namespace).deleteContent(this.jaegerConfigs.pop());
-        }
+        cmdKubeClient().namespace(this.namespace).deleteContent(this.jaegerConfigs);
+        cmdKubeClient().delete(this.certManager);
     }
 
-    private void deployJaegerContent() throws FileNotFoundException {
-        final File folder = new File(this.jaegerOperatorFilesPath);
-        final File[] files = folder.listFiles();
-
-        if (files != null && files.length > 0) {
-            for (final File file : files) {
-                final String yamlContent = TestUtils.setMetadataNamespace(file, this.namespace)
-                    .replace("namespace: \"observability\"", "namespace: \"" + this.namespace + "\"");
-                this.jaegerConfigs.push(yamlContent);
-                LOGGER.info("Creating {} from {}", file.getName(), file.getAbsolutePath());
-                cmdKubeClient(this.namespace).applyContent(yamlContent);
-            }
-        } else {
-            throw new FileNotFoundException("Folder with Jaeger files is empty or doesn't exist");
-        }
+    private void deployJaegerContent() throws IOException {
+        this.certManager = new File(certManagerPath);
+
+        LOGGER.info("Deploying CertManager from {}", certManagerPath);
+        cmdKubeClient("").apply(this.certManager);
```

Comment:
Why you don't pass namespace name or leave it empty to use namespace from kubeClusterResource? You do it a few lines bellow.

Reply:
`certManager` is installed in `kube-system` namespace and one more (don't remember the name).
When you specify the namespace there, it will have conflicts and it will simply not be created.
When you leave it empty, there is still used `namespacedCommand`, so `kubeClient().getNamespace()` is used there.
Because I didn't want to make bigger refactoring, I've just pass empty `String` there. I will add comment why I added it there.